pkg/security/ebpf/c/defs.h

```diff
@@ -239,8 +239,9 @@ struct kevent_t {
 };
 
 struct syscall_t {
-    s64 retval;
-    u64 async; /* TODO: optimize the way we retrieve the async boolean */
+    s64 retval:32,
```

Comment:
What about using `kevent_t.type` instead ? If syscalls returns a 64 bits integers, I would prefer not to shrink the value returned by the kernel

Reply:
You're right, I just updated it, thanks.